app/templates/descriptor/index.html

```diff
@@ -43,7 +43,7 @@ <h2 class="ui header">
                     </thead>
                     <tbody>
                     {% for d in descriptors | sort(attribute='name') %}
-                        <tr onclick="window.location.href = '{{ url_for('descriptor.edit_descriptor', desc_id=d.id) }}';">
```

Comment:
javascript in a separate file, please. Alternatively, just use a href on the name of a descriptor instead of making the whole row clickable (if you don't want to create a JS file for only this)